manifest

```diff
@@ -0,0 +1 @@
+D 2019-07-29T12:17:44
```

Comment:
What's that?

Reply:
Removed in following commit, so this needs squashing so it never appears in the first place.

Reply:
@stgraber ah yes, that was an odd one, I've no idea what these files are or where they came from, but I have a suspicion they came from the static analysis tests as I have recently installed the flake8 tool so they can run and have noticed an unwanted side-effect, every time I run the full test suite now, anything that is currently unstaged in my branch is staged in git after the tests have run, these must have accidentally been committed before I noticed this behaviour was happening. I'll remove.